boards/renesas/ek_ra8m1/doc/index.rst

```diff
@@ -0,0 +1,171 @@
+  providing precision 20.000MHz and 32,768 Hz refeence clocks.
+  Additional low precision clocks are avaialbe internal to the RA8M1 MCU
+
+**System Control and Ecosystem Access**
+
+- USB Full Speed Host and Device (micro-AB connector)
+- Four 5V input sources
+
+  - USB (Debug, Full Speed, High Speed)
+  - External power supply (using surface mount clamp test points and power input vias)
+
+- Three Debug modes
+
+  - Debug on-board (SWD)
+  - Debug in (ETM, SWD and JTAG)
+  - Debug out (SWD)
+
+- User LEDs and buttons
+
+  - Three User LEDs (red, blue, green)
+  - Power LED (white) indicating availability of regulated power
+  - Debug LED (yellow) indicating the debug connection
+  - Two User buttons
+  - One Reset button
+
+- Five most popular ecosystems expansions
+
+  - Two Seeed Grove system (I2C/I3C) connectors
+  - One SparkFun Qwiic connector
+  - Two Digilent Pmod (SPI, UART and I2C/I3C) connectors
+  - Arduino (Uno R3) connector
+  - MikroElektronika mikroBUS connector
+
+- MCU boot configuration jumper
+
+**Special Feature Access**
+
+- Ethernet (RJ45 RMII interface)
+- USB High Speed Host and Device (micro-AB connector)
+- 512 Mb (64 MB) External Octo-SPI Flash (present in the MCU Native Pin Access area of the EK-RA8M1 board)
+- CAN FD (3-pin header)
+
+.. figure:: ek-ra8m1-board.jpg
+	:align: center
+	:alt: RA8M1 Evaluation Kit
+
+	EK-RA8M1 Board Functional Area Definitions (Credit: Renesas Electronics Corporation)
+
+Hardware
+********
+Detail Hardware feature for the RA8M1 MCU group can be found at `RA8M1 Group User's Manual Hardware`_
+
+.. figure:: ra8m1-block-diagram.jpg
+	:width: 442px
+	:align: center
+	:alt: RA8M1 MCU group feature
+
+	RA8M1 Block diagram (Credit: Renesas Electronics Corporation)
+
+Detail Hardware feature for the EK-RA8M1 MCU can be found at `EK-RA8M1 - User's Manual`_
+
+Supported Features
+==================
+
+The below features are currently supported on Zephyr OS for EK-RA8M1 board:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ek_ra8m1`` board configuration can be
+built, flashed, and debugged in the usual way. See
+:ref:`build_an_application` and :ref:`application_run` for more details on
+building and running.
+
+**Note:** Currently Zephyr SDK v0.16.5 hasn't supported for Cortex Arm-M85 build. To build for EK-RA8M1
```

Comment:
Note needs an update to mention that Zephyr SDK v0.16.6 is needed.

Reply:
@ydamigos : Thank you so much for your feedback. We updated it. Please check.